query-engine/connector-test-kit-rs/query-tests-setup/src/runner/mod.rs

```diff
@@ -1,288 +1,258 @@
-mod binary;
-mod direct;
 mod json_adapter;
```

Comment:
All these changes in this file are just moving the logic from the direct runner code?

Reply:
Yes, moving, removing the RunnerInterface, and inlining DirectRunner into Runner